.github/workflows/timeseries-scans-virus-bandit.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
         mkdir -p reports
         docker pull clamav/clamav
         echo "### Virus Scan Results" >> $GITHUB_STEP_SUMMARY
-        docker run --rm -v "${{ github.workspace }}:/src" clamav/clamav clamscan -r  /src/edge-ai-libraries/microservices/time-series-analytics/ > ./reports/clamav-report.txt || true
+        docker run --rm -v "${{ github.workspace }}:/src" clamav/clamav clamscan -r  /src/edge-ai-libraries-repo/microservices/time-series-analytics/ > ./reports/clamav-report.txt || true

```

Comment:
I feel it's good to remove line 61 or stick to edge-ai-libraries there too, so we just use the edge-ai-libraries here just to be consistent.